README.md

```diff
@@ -105,52 +104,73 @@ if err := client.ScaleApplicationInstances(application.ID, 10); err != nil {
 }
 ```
 
-**Subscription & Events**
+### Subscription & Events
+
+Request to listen to events related to applications — namely status updates, health checks
+changes and failures. There are two different event transports controlled by `EventsTransport`
+setting with the following possible values:
+`EVENTS_TRANSPORT_SSE` (see [Event Stream](https://mesosphere.github.io/marathon/docs/rest-api.html#event-stream)) and
+`EVENTS_TRANSPORT_CALLBACK` (default value, see [Event Subscriptions](https://mesosphere.github.io/marathon/docs/rest-api.html#event-subscriptions)).
+The former is available at Marathon >= 0.9.0. The latter requires to start a built-in web server
+accessible by Marathon to connect and push events to. When using Event Subscriptions, consider the following settings:
 
-Request to listen to events related to applications - namely status updates, health checks changes and failures
+- `EventsInterface` — the interface we should be listening on for events. Default `"eth0"`.
+- `EventsPort` — built-in web server port. Default `10001`.
 
 ```Go
-/* step: lets register for events */
-update := make(marathon.EventsChannel,5)
-if err := client.AddEventsListener(update, marathon.EVENTS_APPLICATIONS); err != nil {
-	glog.Fatalf("Failed to register for subscriptions, %s", err)
+// configure client
+config := marathon.NewDefaultConfig()
+config.URL = marathon_url
+config.EventsTransport = marathon.EVENTS_TRANSPORT_SSE
+
+if client, err := marathon.NewClient(config); err != nil {
+	glog.Fatalf("Failed to create a client for marathon, error: %s", err)
 } else {
-	for {
-	    event := <-update
-	    glog.Infof("EVENT: %s", event )
+	// lets register for events
+	update := make(marathon.EventsChannel, 5)
+	if err := client.AddEventsListener(update, marathon.EVENTS_APPLICATIONS); err != nil {
+		glog.Fatalf("Failed to register for subscriptions, %s", err)
+	} else {
+		for {
+			event := <-update
+			glog.Infof("EVENT: %s", event)
+		}
```

Comment:
Since you're touching the examples anyway, what do you think about removing `else` statements from them?